### ISSUE TITLE
#1060792779------------- Buttons must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1334]
<strong>Source:</strong>`<button aria-label="" aria-live="polite" class="button-padding--shrunk icon-end-empty" type="button"><calcite-icon class="icon icon--start" aria-hidden="true" icon="zoomToObject" scale="s" calcite-hydrated=""></calcite-icon></button>`

<strong>Element Location: </strong>[".p-selectable-row[data-p-highlight=\"false\"][data-p-selectable-row=\"true\"]:nth-child(18) > .ui-resizable-column.textOverflow:nth-child(8) > .c-coordinateJobPanel__gridActionBtn[icon-start=\"zoomToObject\"][kind=\"neutral\"]",".button-padding--shrunk.icon-end-empty[aria-label=\"\"]"]

<strong>Impact: </strong>critical

<strong>Failure Summary:</strong> Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,section508,section508.22.a,TTv5,TT6.a,EN-301-549,EN-9.4.1.2,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.8/button-name?application=axeAPI
